### ISSUE TITLE
RFC4: Staff

### ISSUE THREAD

**Issue by k8hertweck** [carpentries/2017Merger#4]
View RFC4 [here](https://github.com/carpentries/2017Merger/blob/master/RFCs/RFC4.md)

**Q1: Do you have any comments or suggestions on the role of staff in the context of The Carpentries (umbrella organisation)?**

**Q2: What do you see as the most urgent staffing priorities for The Carpentries?**

**Comment by JasonJWilliamsNY** [comment]
Q2: I would bet that everything specified so far is great, but that it leaves everyone's hands full. I think it will probably work for a while, but what I don't see is innovation. What is the path forward for Carpentries beyond 2-day workshops? It's possible community effort will innovate this, and all the staff are bright, creative (and busy). A curriculum manager (From the world of open education resources?) could liaise with lesson orgs to move Carpentries into new teaching formats and modalities.  There are ways this can happen without a person directly driving, but having that person may actually make it happen.

**Comment by pitviper6** [comment]
My only thought is that the Carpentries are exploding, and make sure that there are enough of your excellent staff to prevent burnout!

**Comment by jduckles** [comment]
I've always wanted to find a balance with the staff to catalyze a spirit of volunteerism where the staff's effort is focused on lowering impediments to volunteering. We do strike a reasonable balance at this, but there are things that are hard that we can make easier. We're trying as a staff to find ways to not grow staff linearly with lessons/memberships/workshops, but find ways to scale our impact. With the Carpentries umbrella structure we can continue to realize increasing efficiencies as we grow. 

Innovations are taking place beyond two-day workshops in how we enable our member organizations to realize benefits of an open and safe workshop culture and expand their own local impact on researcher practices beyond the workshop. This is essential to building the foundation for the kind of change in practice we're championing with workshops. In some ways I see this part as an "Industry association" of like-minded research organizations that are working to improve computational research practices. Workshops are a part of this, but with our membership program we've begun a slow pivot to catalyzing workshops and other activities at local campuses that teach and empower via multiple modalities.

**Comment by ChristinaLK** [comment]
+1 to Jonah's comment about catalyzing volunteerism + continuing to invest in/encourage the community impact beyond workshops.

**Comment by npch** [comment]
I would love to see a staff member (probably under the general "community engagement" banner) focusing on helping member organisations setup internal mechanisms for continuing support for learning and development (e.g. code clubs, drop-in sessions, hack sessions...). These would be resourced by the organisations, but the staff member would be there to provide support, encouragement and guidance in best practice.

**Comment by gvwilson** [comment]
I agree with @npch - if I could wind the clock back to 2012, I would probably have pursued a regional chapter model for growth, and I think that someone devoted to this now could do a world of good.

**Comment by gvwilson** [comment]
Separately, should this RFC spell out who decides what new staff positions to create? For example, if the ED decides that the org should hire someone for sys admin/tech support, does she then go ahead and do it (budget permitting), or does the position need to be approved by the Board first?

**Comment by npch** [comment]
Adding to @gvwilson comment, I think that there needs to be a way of distinguishing between "senior staff" and "other staff" recruitment. Often this is split by understanding which staff are at Cxx or xD level - so the hire of a new Director for Novel and Amazing Hats would be passed by the board, but (assuming that the current plan and roadmap includes a project to improve the headgear of instructors, and a budget has been passed) hiring of an appropriate Better Hats Coordinator would be in the remit of the ED.

I'm assuming that most new staff positions will be suggested and created by the ED and senior staff team, based on the plans and budgets approved by the Board.
